elkm1_lib/counters.py

```diff
@@ -41,7 +41,7 @@ def _got_desc(self, descriptions: list[str | None], desc_type: int) -> None:
         # Only poll counters that have a name defined
         for counter in self.elements:
             if not counter.is_default_name():
-                self._connection.send(cv_encode(counter.index))
+                self._connection.send(cv_encode(counter.index), True)
```

Comment:
```suggestion
                self._connection.send(cv_encode(counter.index), priority_send = True)
```

I'd probably use a kwarg with a boolean for anyone else reading this